connectivity/drivers/ble/FEATURE_BLE/COMPONENT_CYW43XXX/firmware/TARGET_CYW9P62S1_43012EVB_01/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-target_sources(mbed-ble
+add_library(mbed-ble-cyw9p62s1_43012evb_01 INTERFACE)
```

Comment:
This CMake library is not referenced in any Mbed target. Should it?

Reply:
Yes, this should be a new issue. I'll report it.